core/modules/contact/contact.module

```diff
@@ -198,6 +215,18 @@ function contact_config_data() {
   return $config_data;
 }
 
+/**
+ * Show the personnal copy checkbox on forms
```

Comment:
The word "personal" only has one "n", and the sentence should end with a period.
I think this wording better defines the function: Determines if the 'Send yourself a copy' checkbox should be shown.